lib/attr_encrypted.rb

```diff
@@ -277,6 +310,7 @@ module InstanceMethods
     #  @user = User.new('some-secret-key')
     #  @user.decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
     def decrypt(attribute, encrypted_value)
+      self.class.encrypted_attributes[attribute.to_sym][:operation] = :decrypting
```

Comment:
I'm not sure what the Concurrent::Hash gives us other than another gem dependency. Do you mind explaining what you were thinking?

Also, I very much agree that we need to stop mutating shared state. Please stand by for another commit where I introduce the encrypted_attributes function at the instance level. In part to address this: https://github.com/attr-encrypted/attr_encrypted/pull/182/files#diff-88607ee1fc6781577944dc73a3630d96R333